inst/sql/sql_server/CreateResultsDataModel.sql

```diff
@@ -31,95 +37,42 @@
-
-IF OBJECT_ID('@results_schema.@concept_sets', 'U') IS NOT NULL
-    DROP TABLE @results_schema.@concept_sets;
-
-IF OBJECT_ID('@results_schema.@concept_synonym', 'U') IS NOT NULL
-    DROP TABLE @results_schema.@concept_synonym;
-
-IF OBJECT_ID('@results_schema.@covariate_ref', 'U') IS NOT NULL
-    DROP TABLE @results_schema.@covariate_ref;
-
-IF OBJECT_ID('@results_schema.@covariate_value', 'U') IS NOT NULL
-    DROP TABLE @results_schema.@covariate_value;
-
-IF OBJECT_ID('@results_schema.@covariate_value_dist', 'U') IS NOT NULL
-    DROP TABLE @results_schema.@covariate_value_dist;
-
-IF OBJECT_ID('@results_schema.@database', 'U') IS NOT NULL
-    DROP TABLE @results_schema.@database;
-
-IF OBJECT_ID('@results_schema.@domain', 'U') IS NOT NULL
-    DROP TABLE @results_schema.@domain;
-
-IF OBJECT_ID('@results_schema.@incidence_rate', 'U') IS NOT NULL
-    DROP TABLE @results_schema.@incidence_rate;
-
-IF OBJECT_ID('@results_schema.@included_source_concept', 'U') IS NOT NULL
-    DROP TABLE @results_schema.@included_source_concept;
-
-IF OBJECT_ID('@results_schema.@inclusion_rule_stats', 'U') IS NOT NULL
-    DROP TABLE @results_schema.@inclusion_rule_stats;
-
-IF OBJECT_ID('@results_schema.@index_event_breakdown', 'U') IS NOT NULL
-    DROP TABLE @results_schema.@index_event_breakdown;
-
-IF OBJECT_ID('@results_schema.@metadata', 'U') IS NOT NULL
-    DROP TABLE @results_schema.@metadata;
-
-IF OBJECT_ID('@results_schema.@orphan_concept', 'U') IS NOT NULL
-    DROP TABLE @results_schema.@orphan_concept;
-
-IF OBJECT_ID('@results_schema.@phenotype_description', 'U') IS NOT NULL
-    DROP TABLE @results_schema.@phenotype_description;
-
-IF OBJECT_ID('@results_schema.@relationship', 'U') IS NOT NULL
-    DROP TABLE @results_schema.@relationship;
-
-IF OBJECT_ID('@results_schema.@resolved_concepts', 'U') IS NOT NULL
-    DROP TABLE @results_schema.@resolved_concepts;
-
-IF OBJECT_ID('@results_schema.@temporal_analysis_ref', 'U') IS NOT NULL
-    DROP TABLE @results_schema.@temporal_analysis_ref;
-
-IF OBJECT_ID('@results_schema.@temporal_covariate_ref', 'U') IS NOT NULL
-    DROP TABLE @results_schema.@temporal_covariate_ref;
-
-IF OBJECT_ID('@results_schema.@temporal_covariate_value', 'U') IS NOT NULL
-    DROP TABLE @results_schema.@temporal_covariate_value;
-
-IF OBJECT_ID('@results_schema.@temporal_covariate_value_dist', 'U') IS NOT NULL
-    DROP TABLE @results_schema.@temporal_covariate_value_dist;
-
-IF OBJECT_ID('@results_schema.@temporal_time_ref', 'U') IS NOT NULL
-    DROP TABLE @results_schema.@temporal_time_ref;
-
-IF OBJECT_ID('@results_schema.@time_distribution', 'U') IS NOT NULL
-    DROP TABLE @results_schema.@time_distribution;
-
-IF OBJECT_ID('@results_schema.@visit_context', 'U') IS NOT NULL
-    DROP TABLE @results_schema.@visit_context;
-
-IF OBJECT_ID('@results_schema.@vocabulary', 'U') IS NOT NULL
-    DROP TABLE @results_schema.@vocabulary;
+DROP TABLE IF EXISTS @results_schema.@cohort;
```

Comment:
This is postgres specific syntax and will break most databasae platforms. Its also not related to the scope of the PR.

Reply:
Martijn dropped this new knowledge for me - its supported sqlRender 1.9+

![image](https://user-images.githubusercontent.com/13936600/156254727-f78d513a-b54a-47e0-b817-dfda7aebf8d9.png)